docs/tutorials/first-program.md

```diff
@@ -78,12 +78,13 @@ When you added the service, Ballerina configured the resource to use the GET met
 
 You can click the symbol again to confirm that GET was in fact changed to POST. You can also click the **Source View** button in the lower right corner to see the changes that are being made to the Ballerina code as you work with the visual editor. Currently, the source code should look like this:
 
-```
+```Ballerina
 @http:BasePath{value:"/myecho"} 
+
 service myEchoService {
-	@http:POST {} 
-	resource myEchoResource( message m) {
-	}
+    @http:POST {} 
+        resource myEchoResource( message m) {
```

Comment:
Can't we move to the new API? Any restriction on that?

Reply:
Need to update whole doc since it has references to lots of removed content in the language. I was just focusing on highlighting code snippets only :) I'll update the content to some extent as well.